### ISSUE TITLE
Should we add a column in booster table for list of cards in the pack?

### ISSUE THREAD

**Issue by MeLikeChoco** [chinhodado/YgoSqliteDbCreator#3]
Once archetype and raw text are finished and approved, we could extend the table Booster with a column that holds a list of cards in the booster pack. 

Although I see your app having the ability to display card list, do you process that in the app?

**Comment by chinhodado** [comment]
Yeah I guess that can be added to the database. Right now the app fetches that info directly from wikia.

**Comment by MeLikeChoco** [comment]
Is this character a good delimiter? "|"

**Comment by chinhodado** [comment]
Depending on the data, but yes in general it is a good delimiter.
